cmd/push.go

```diff
@@ -302,7 +303,7 @@ func buildImage(ctx context.Context, dev *model.Dev, imageFromApp string, pushOp
 		BuildArgs:  buildArgs,
 		OutputMode: pushOpts.Progress,
 	}
-	if err := buildv1.NewBuilderFromScratch().Build(ctx, buildOptions); err != nil {
+	if err := buildv1.NewBuilderFromScratch(io.NewIOController()).Build(ctx, buildOptions); err != nil {
```

Comment:
should this be injected? or at least add a TODO comment to remember inject it when we address this command refactor